hack/k8s/helm/nuclio/values.yaml

```diff
@@ -59,6 +62,12 @@ dashboard:
     #    hosts:
     #      - nuclio.local
 
+kaniko:
+  image:
+    repository: gcr.io/kaniko-project/executor
```

Comment:
`insecurePushRegistry`, `insecurePullRegistry` and `cacheRepo` values should be moved here as well

Reply:
👍  Also moving this entire block under dashboard